CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 0.22.0-rc.0 / 2021-05-04
+
+* [CHANGE] Amtool and Alertmanager binaries help now prints to stdout. #2505
+* [CHANGE] Use path relative to the configuration file for certificates and password files. #2502
+* [CHANGE] Display Silence and Alert dates in ISO8601 format. #2363
+* [FEATURE] Add date picker to silence form views. #2262
+* [FEATURE] Add support for negative matchers. #2434 #2460 and many more.
+* [FEATURE] Add time-based muting to routing tree. #2393
+* [FEATURE] Support TLS and basic authentication on the web server. #2446
+* [FEATURE] Add OAuth 2.0 client support in HTTP client. #2560
+* [ENHANCEMENT] Add composite durations in the configuration (e.g. 2h20m). #2353
+* [ENHANCEMENT] Add follow_redirect option to disable following redirects. #2551
+* [ENHANCEMENT] Add metric for permanently failed notifications. #2383
+* [ENHANCEMENT] Add support for custom authorization scheme. #2499
+* [ENHANCEMENT] Add support for not following HTTP redirects. #2499
+* [ENHANCEMENT] Add support to set the Slack URL from a file. #2534
+* [ENHANCEMENT] amtool: Add alert status for extended and simple output. #2324
```

Comment:
I had difficulty understanding what this is about. Perhaps `for` → `to` would make it clearer?